src/cpu/core_dynrec/decoder_basic.h

```diff
@@ -292,9 +292,14 @@ static void INLINE decode_increase_wmapmask(Bitu size) {
 	// update mask entries
 	switch (size) {
 		case 1 : activecb->cache.wmapmask[mapidx]+=0x01; break;
-		case 2 : (*(Bit16u*)&activecb->cache.wmapmask[mapidx])+=0x0101; break;
-		case 4 : (*(Bit32u*)&activecb->cache.wmapmask[mapidx])+=0x01010101; break;
-	}
+	        case 2:
```

Comment:
I think the whole switch can be reformatted here - this will probably put cases in the same column as `switch` keyword and be consistent with the previous switch in here.

Reply:
Just as you expected; much better.